cognite/neat/rules/models/_rules/dms_architect_rules.py

```diff
@@ -0,0 +1,115 @@
+        ...
+del subclasses  # cleanup namespace
+
+
+class DMSArchitectMetadata(BaseMetadata):
+    role: ClassVar[RoleTypes] = RoleTypes.dms_architect
+    space: str
+    external_id: str = Field(alias="externalId")
+    version: str | None = Field(
+        description="Data model version",
+        min_length=1,
+        max_length=43,
+    )
+
+    contributor: str | list[str] = Field(
+        description=(
+            "List of contributors to the data model creation, "
+            "typically information architects are considered as contributors."
+        ),
+    )
+
+    @classmethod
+    def from_information_architect_metadata(
+        cls, metadata: InformationMetadata, space: str | None = None, externalId: str | None = None
+    ):
+        metadata_as_dict = metadata.model_dump()
+        metadata_as_dict["space"] = space or "neat-playground"
+        metadata_as_dict["externalId"] = externalId or "neat_model"
+        return cls(**metadata_as_dict)
+
+    @classmethod
+    def from_domain_expert_metadata(
+        cls,
+        metadata: DomainMetadata,
+        space: str | None = None,
+        external_id: str | None = None,
+        version: str | None = None,
+        contributor: str | list[str] | None = None,
+        created: datetime | None = None,
+        updated: datetime | None = None,
+    ):
+        information = InformationMetadata.from_domain_expert_metadata(
+            metadata, None, None, version, contributor, created, updated
+        ).model_dump()
+
+        return cls.from_information_architect_metadata(information, space, external_id)
+
+
+class DMSProperty(SheetEntity):
+    class_: str = Field(alias="Class")
+    property: str = Field(alias="Property")
+    description: str | None = Field(None, alias="Description")
+    value_type: type[PropertyType] | str = Field(alias="Value Type")
+    nullable: bool | None = Field(default=None, alias="Nullable")
+    is_list: bool | None = Field(default=None, alias="IsList")
+    default: str | None = Field(None, alias="Default")
+    source: str | None = Field(None, alias="Source")
+    container: str | None = Field(None, alias="Container")
+    container_property: str | None = Field(None, alias="ContainerProperty")
+    view: str | None = Field(None, alias="View")
+    view_property: str | None = Field(None, alias="ViewProperty")
+    index: str | None = Field(None, alias="Index")
+    constraint: str | None = Field(None, alias="Constraint")
+
+    @field_validator("value_type", mode="before")
+    def to_type(cls, value: Any) -> Any:
+        if isinstance(value, str) and value.casefold() in _PropertyType_by_name:
+            return _PropertyType_by_name[value.casefold()]
+        return value
+
+
+class DMSContainer(SheetEntity):
+    container: str = Field(alias="Container")
+    description: str | None = Field(None, alias="Description")
+    constraint: str | None = Field(None, alias="Constraint")
+
+
+class DMSView(SheetEntity):
```

Comment:
You might want to do the same thing you did with the `Properties` sheet here, ie. renaming `Classes` sheet to `Views` and adding these additional columns